modules/home-manager/options/misc.nix

```diff
@@ -48,5 +48,10 @@ in {
         with the canvas (practically anything at this point)
       '';
     };
+
+    firefoxSync = mkEnableOption ''
+      Enable Firefox Sync. Disabled by default to prevent calling home.
```

Comment:
`mkEnableOption` [implies "Whether to enable..."](https://github.com/hsjobeki/nixpkgs/blob/migrate-doc-comments/lib/options.nix#L142) in resulting documentation. This should be just `Firefox Sync."